README.md

```diff
@@ -976,6 +976,283 @@ Group membership can be created by:
   $group->addAccount($account);
   ```
 
+### Integrating with Google
+
+Stormpath supports accessing accounts from a number of different 
+locations including Google. Google uses OAuth 2.0 protocol for 
+authentication / authorization and Stormpath can leverage their 
+authorization code (or access tokens) to return an `Account` for 
+a given code.
+
+The steps to enable this functionality into your application include:
+
++ Create a Google Directory
++ Create an `AccountStoreMapping` between a Google Directory and your `Application`
++ Accessing Accounts with Google Authorization Codes or Access Tokens
+
+Google Directories follow behavior similar to mirror directories, but 
+have a `Provider` resource that contains information regarding the Google 
+application that the directory is configured for.
+
+#### Google Provider Resource
+
+A GoogleProvider resource holds specific information needed for working with 
+a Google Directory. It is important to understand the format of the provider 
+resource when creating and updating a Google Directory.
+
+A provider resource can be obtained by accessing the directory’s provider as 
+follows:
+
+```PHP
+$provider = $client->dataStore->getResource("https://api.stormpath.com/v1/directories/1mBDmVgW8JEon4AkoSYjPv/provider",
+    \Stormpath\Stormpath::GOOGLE_PROVIDER);
+```
```

Comment:
Should also provide the documentation to make this call using the static client configuration. In this case, the call would look something like:

```php
// The user should also have the option to specify the URL with "/provider", 
// or the partial path (which could be "directories/DIR_ID/provider", 
// or "DIR_ID/provider" or just "DIR_ID"). 
$directoryHref = "https://api.stormpath.com/v1/directories/1mBDmVgW8JEon4AkoSYjPv"; 
$provider = GoogleProvider::get($directoryHref);
```
I noticed that the new resource provider classes don't have the applicable static methods (like the `get` I just mentioned), so please add them to these classes. See how this is implemented in the [Account class for reference](https://github.com/stormpath/stormpath-sdk-php/blob/master/src/Stormpath/Resource/Account.php).